<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewItem</class>
 <widget class="QDialog" name="NewItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblCreateTitle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create a new item</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblItemName">
      <property name="text">
       <string>Item name: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditItemName">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Type name company or service</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblUser">
      <property name="text">
       <string>User:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditUser"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblPassword">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditPassword"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblUrl">
      <property name="text">
       <string>URL:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditUrl"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblCategory">
      <property name="text">
       <string>Category:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="cmbBoxCategory"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lblItemDescription">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="textEditItemDescription">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="btnSaveClose">
      <property name="text">
       <string>Save and close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
